tests/config_test.py

```diff
@@ -129,3 +137,13 @@ def testConfigOverrideFlag(self):
     config = configs.GetUserConfig()
     self.assertEqual(config['a']['vm_groups']['default']['vm_count'], 5)
     self.assertEqual(config['a']['flags']['flag'], 'value')
+
+  def testConfigFlagStillParsed(self):
+    flags.DEFINE_list('list_flag', ['default_val'], 'List flag.')
+    list_flag_config = yaml.load(LIST_FLAG_CONFIG)
+    flag_values = configs.GetMergedFlags(list_flag_config['a'])
+
+    self.assertEqual(flag_values.list_flag,
+                     ['a', 'b', 'c'])
+
+    del FLAGS.list_flag
```

Comment:
RE: previous comment, the reason is here. This cleanup statement doesn't get called if a prior step of the test (like the assert) fails.